cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/OperatorsTest.scala

```diff
@@ -245,9 +253,96 @@ class OperatorsTest extends DocumentingTest {
+            //CYPHER_TODO
+          })) {
+          resultTable()
+        }
+        p(
+          """<<cypher-temporal-duration-component, Components of a _Duration_>> that do not apply to the temporal instant are ignored.
+            |For example, when adding a _Duration_ to a _Date_, the _hours_, _minutes_, _seconds_ and _milliseconds_ of the _Duration_ are ignored (_Time_ behaves in an analogous manner):
+          """.stripMargin)
+        query(
+          """WITH date({year:1984, month:10, day:11}) AS aDate
+            |UNWIND [duration({years: 12, months: 5, days: 14, hours: 16, minutes: 12, seconds: 70, nanoseconds: 2}),
+            |   duration({months:1, days: -14, hours: 16, minutes: -12, seconds: 70}),
+            |   duration({years: 12.5, months: 5.5, days: 14.5, hours: 16.5, minutes: 12.5, seconds: 70.5, nanoseconds: 3})] AS aDuration
+            |RETURN aDate + aDuration, aDate - aDuration""".stripMargin, ResultAssertions((r) => {
+            //CYPHER_TODO
+          })) {
+          resultTable()
+        }
+        p(
+          """Adding two durations to a temporal instant is not an associative operation.
+            |This is because non-existing dates are truncated to the nearest existing date:""".stripMargin)
+        query(
+          """RETURN (date("2011-01-31") + duration("P1M")) + duration("P12M") AS date1,
+            |   date("2011-01-31") + (duration("P1M") + duration("P12M")) AS date2
+          """.stripMargin, ResultAssertions((r) => {
+            //CYPHER_TODO date1: date({year:2012, month:2, day:28}); date2: date({year:2012, month:2, day:29})
+          })) {
+          resultTable()
+        }
+      }
+      section("Adding and subtracting a _Duration_ to or from another _Duration_", "syntax-add-subtract-duration-to-duration") {
+        query(
+          """WITH [duration({years: 12, months: 5, days: 14, hours: 16, minutes: 12, seconds: 70, nanoseconds: 1}),
+            |   duration({months:1, days: -14, hours: 16, minutes: -12, seconds: 70})] AS list
+            |UNWIND list AS duration1
+            |UNWIND list AS duration2
+            |RETURN duration1 + duration2, duration1 - duration2""".stripMargin, ResultAssertions((r) => {
+            //CYPHER_TODO
+          })) {
+          resultTable()
+        }
+      }
+      section("Multiplying and dividing a _Duration_ with or by a number", "syntax-multiply-divide-duration-number") {
+        p("""These operations are interpreted simply as component-wise operations with overflow to smaller units based on an average length of units in the case of division (and multiplication with fractions).""".stripMargin)
+        query(
+          """WITH duration({years: 12, months: 5, days: 14, hours: 16, minutes: 12, seconds: 70, nanoseconds: 1}) AS aDuration
+            |UNWIND [1, 2, 0.5] as aNumber
+            |RETURN aDuration * aNumber, aDuration/aNumber""".stripMargin, ResultAssertions((r) => {
```

Comment:
I think the queries should return the original number and duration fields, otherwise it is hard to understand the results.